test/python/pulse/test_instructions.py

```diff
@@ -49,26 +56,37 @@ def test_can_construct_valid_acquire_command(self):
         self.assertEqual(acq.kernel.params, kernel_opts)
         self.assertIsInstance(acq.id, int)
         self.assertEqual(acq.name, 'acquire')
-        self.assertEqual(acq.operands, (10, AcquireChannel(0), MemorySlot(0), None))
+        self.assertEqual(acq.operands, (10,
+                                        channels.AcquireChannel(0),
+                                        channels.MemorySlot(0), None))
 
     def test_isntructions_hash(self):
```

Comment:
```suggestion
    def test_instructions_hash(self):
```
Ik this is out of scope, up to you.

Reply:
Fixed in 8f1ad270f